db/migrations/20230103113049_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  email VARCHAR(200) NOT NULL,
+  password VARCHAR(200) NOT NULL,
+  name VARCHAR(200) NOT NULL,
+  phoneNumber VARCHAR(200) NOT NULL,
```

Comment:
```suggestion
  phone_number VARCHAR(20) NOT NULL,
```
database 관련 컨벤션에는 camelCase를 쓰지 않습니다. 또한 phone_number를 저장하는 field에는 200만큼 메모리를 할당할 필요가 없을테니 적절하게 수정해주세요~